.github/workflows/timeseries-scans-dbs.yaml

```diff
@@ -5,58 +5,68 @@
 #
 name: "[Time SeriesAnalytics] Docker Bench Security Scan"
 run-name: "[Time Series Analytics] Docker Bench Security Scan workflow (by @${{ github.actor }} via ${{ github.event_name }})"
-permissions: read-all
 
 on:
   workflow_dispatch:
   
 jobs:
   DBS_job:
     runs-on: ubuntu-24.04
-    
+    permissions:
+      contents: read
+      packages: read          # needed for actions/checkout
     steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
       - name: check the system
         run: |
           docker ps &&
           uname -a &&
           docker version && 
           git version && 
           docker compose version
+
       - name: Runner workspace path
         run: |
-           echo "Cleaning up previous run"
-           if [ -n "$(docker ps -aq)" ]; then
-               docker stop $(docker ps -aq)
-           fi
-           if [ -n "$(docker ps -aq)" ]; then
-               docker rm $(docker ps -aq)
-           fi
-           docker images --quiet --filter=dangling=true | xargs --no-run-if-empty docker rmi -f
-           echo $PASSWORD | sudo -SE rm -rf "${{ github.workspace }}"
+          echo "Cleaning up previous run"
+          if [ -n "$(docker ps -aq)" ]; then
+              docker stop $(docker ps -aq)
+          fi
+          if [ -n "$(docker ps -aq)" ]; then
+              docker rm $(docker ps -aq)
+          fi
+          docker images --quiet --filter=dangling=true | xargs --no-run-if-empty docker rmi -f

```

Comment:
I feel lines 33-40 aren't required becz everytime we run the job I believe we get a new VM. If we still wants to retain if the same VM is getting assigned, the line 40 is not needed.